### ISSUE TITLE
Lots of __pycache__ entries in the map files

### ISSUE THREAD

**Issue by duncanmmacleod** [conda-incubator/conda-suggest#4]
At least on windows, there are lots of `__pycache__` entries in the map files:

```
❯ bash -c "grep __pycache__ /mnt/c/Users/spxdmm/Miniconda3/envs/conda-suggest/share/conda-suggest/conda-forge.win-64.map"
__pycache__:aospy
__pycache__:arcsi
__pycache__:aubio
__pycache__:caiman
__pycache__:cis
__pycache__:compliance-checker
__pycache__:conda
__pycache__:conda-build-all
__pycache__:constructor
__pycache__:cookiecutter
__pycache__:coverage
__pycache__:cssutils
__pycache__:django
__pycache__:docutils
__pycache__:dodgy
__pycache__:dolfyn
__pycache__:esys-escript
__pycache__:exec-wrappers
__pycache__:fastavro
__pycache__:gensim
__pycache__:ispyb
__pycache__:jupyter_core
__pycache__:legit
__pycache__:mdshare
__pycache__:meson
__pycache__:numpy
__pycache__:obspy
__pycache__:pbr
__pycache__:pdfminer.six
__pycache__:peewee
__pycache__:pillow
__pycache__:ps2ff
__pycache__:pyamg
__pycache__:pychemia
__pycache__:pyemd
__pycache__:pygments
__pycache__:pymapd
__pycache__:pymca
__pycache__:pymor
__pycache__:pyresttest
__pycache__:python
__pycache__:python-fmask
__pycache__:pytide
__pycache__:pyuvdata
__pycache__:pyvcf
__pycache__:qds-sdk
__pycache__:safer
__pycache__:seapy
__pycache__:setuptools
__pycache__:spglib
__pycache__:sphinx-gallery
__pycache__:spylon
__pycache__:statuspage
__pycache__:tlslite
__pycache__:vulture
__pycache__:websocket-client
__pycache__:wordcloud
```

It might be just as simple as to blacklist that name from the executable discovery logic.

**Comment by scopatz** [comment]
Ahh good catch, thanks... yeah, I'll filter these out.

**Comment by scopatz** [comment]
This has been implemented in f24db76
